code/modules/mob/dead/new_player/new_player.dm

```diff
@@ -400,9 +400,11 @@
 			switch(SSshuttle.emergency.mode)
 				if(SHUTTLE_RECALL, SHUTTLE_IDLE)
 					SSticker.mode.make_antag_chance(humanc)
+					SSticker.mode.make_special_antag_chance(humanc)
 				if(SHUTTLE_CALL)
 					if(SSshuttle.emergency.timeLeft(1) > initial(SSshuttle.emergencyCallTime)*0.5)
 						SSticker.mode.make_antag_chance(humanc)
+						SSticker.mode.make_special_antag_chance(humanc)

```

Comment:
Can you explain why you're calling this here?

Reply:
For people who join the shift late